subprojects/snapshots/snapshots.gradle.kts

```diff
@@ -62,3 +64,16 @@ gradlebuildJava {
     moduleType = ModuleType.CORE
 }
 
+afterEvaluate {
+    // This is a workaround for the validate plugins task trying to inspect classes which
+    // have changed but are NOT tasks
```

Comment:
IIUC the workaround is to exclude `internal` packages. Can you add a comment saying that? Or maybe rename the method `isInternal()` to something with more context, like `isInInternalPackageOf()`?

Reply:
I'm also unsure how useful adding two methods on `File` is better than actually writing out the code in the `filter { /* ... */ }` block. But I guess that's a matter of taste.

Reply:
I am using a pattern on the relative path now, so this is now much easier to read.